app/controllers/api/v1/admin/communications_controller.rb

```diff
@@ -18,6 +18,8 @@ def create
         end
 
         def update
+          raise ActionController::BadRequest, 'can not update a published communication' if @communication.published
```

Comment:
Cannot is a single word, on the other hand, see my PR, this should be fixed at the model layer, not in the controller.